sdk/go/common/util/cmdutil/testdata/processtree.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016-2022, Pulumi Corporation.
```

Comment:
I might need to rename it something other than .go so it does not compile with the main go tree.

Reply:
Do we not have other .go files under testdata folders?

Reply:
`testdata` should be ignored by the `go` tool, so this shouldn't build by default. Are you running into issues here?

Reply:
Ah nice. Confirmed, it's good.